conjure-java-core/src/test/java/com/palantir/conjure/java/parameterized/TestCases.java

```diff
@@ -176,7 +176,10 @@ public final class TestCases {
                     .files(Path.of("ete-service.yml"))
                     .files(Path.of("ete-binary.yml"))
                     .files(Path.of("example-errors.yml"))
-                    .options(Options.builder().apiVersion("1.2.3").build())
+                    .options(Options.builder()
+                            .apiVersion("1.2.3")
+                            .generateErrorParameterFormatRespectingDialogueInterfaces(true)
```

Comment:
By enabling this flag here, instead of creating a new test case, we are no longer testing the code gen of the current Dialogue interfaces with this test.

I will add a separate test case before merging. I didn't want to do this while we're initially reviewing and cutting RCs: there are loads of files that are generated, which makes reviewing a bit more tedious.